File: ex22_0817/src/main/java/org/zerock/ex22_0817/entity/Memo.java
package org.zerock.ex22_0817.entity;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name= "tb4_memo")
@ToString
@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Memo {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long mno;

    @Column(length = 200, nullable = false)
    private String memoText;
}

File: demo9/src/main/java/com/example/demo9/entity/Memo.kt
package com.example.demo9.entity

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Getter
import lombok.NoArgsConstructor
import lombok.ToString
import org.hibernate.annotations.OnDelete
import javax.persistence.Entity
import javax.persistence.Table

@Entity
@Table(name="tbl_memoo")
@ToString
@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
class Memo {
    @OnDelete
    ;

}